### PR TITLE
Support for console.warn and console.error added

### PR DESCRIPTION
**Pull request by vivekannan**

### CODE REVIEW

File: screenlog.js [hunk 1]
```diff
@@ -1,69 +1,104 @@
...
+	function warn() {
+		return genericLogger(_options.warnColor).apply(null, arguments);
+	}
+	
+	function error() {
+		return genericLogger(_options.errorColor).apply(null, arguments);
+	}
+	
+	function setOptions(options) {
+		for(var i in options)
```
Comment (chinchang) [inline screenlog.js:67]: Needs missing curly brackets. In the following `if` too.

File: screenlog.js [hunk 2]
```diff
@@ -1,69 +1,104 @@
...
+				(logEl.children.length % 2 ? 'rgba(255,255,255,0.1)' : '') + ';color:' + color); // zebra lines
+			var val = [].slice.call(arguments).reduce(function(prev, arg) {
+				return prev + ' ' + (typeof arg === "object" ? JSON.stringify(arg) : arg);
+			}, '');
+			el.textContent = val;
+			
+			logEl.appendChild(el);
+			
+			// Scroll to last element, if autoScroll option is set.
+			if(_options.autoScroll) logEl.scrollTop = logEl.scrollHeight - logEl.clientHeight;
```
Comment (chinchang) [inline screenlog.js]: Needs missing curly brackets

**Comment by chinchang** [comment]
fixs #6

File: example.html [hunk 3]
```diff
@@ -5,21 +5,23 @@
...
-	screenLog.log('Objects', JSON.stringify({a: 3}));
+	screenLog.log('Objects', {a: 3});
 
 	console.log('console.log also gets logged.')
 
 	var i = 10;
 	function log() {
 		console.log('Future log', Date.now());
+		console.error('Future log', Date.now());
+		console.warn('Future log', Date.now());
```
Comment (chinchang) [inline example.html]: To differentiate from log, Let rename from `Future log` to `Future warn`?

File: screenlog.js [hunk 4]
```diff
@@ -1,69 +1,104 @@
...
-
+	
 	var logEl,
 		isInitialized = false,
 		_console = {}; // backup console obj to contain references of overridden fns.
-
-	function createElement( tag, css ) {
-		var element = document.createElement( tag );
+		_options = {
+						bgColor: 'black',
```
Comment (chinchang) [inline screenlog.js]: too much indentation

File: screenlog.js [hunk 5]
```diff
@@ -1,69 +1,104 @@
...
+	
+	function log() {
+		return genericLogger(_options.logColor).apply(null, arguments);
+	}
+	
+	function warn() {
+		return genericLogger(_options.warnColor).apply(null, arguments);
+	}
+	
+	function error() {
```
Comment (chinchang) [inline screenlog.js:62]: @vivekannan While you are at this, please also add support for `console.info` (blue color) to completely fix issue #6

**Comment by chinchang** [comment]
Nice stuff! 👍

**Comment by chinchang** [comment]
Thanks @vivekannan :)

**Comment by vivekannan** [comment]
~~My first pull. YEAH! :)~~
Actually this is my second pull.
Onset of Alzheimer's, YEAH!

**Comment by vivekannan** [comment]
I think something got broke after this merge. Screenlog is no longer working properly for me in Firefox 47.0 and Chrome 51. Everything is getting logged in the actual console, but nothing shows up on the on-screen div.

Edit 1: Also, I pulled your master after the merge, I checked everything before making the pull request.
Edit 2: dist/screenlog.js works perfectly.
Edit 3: Not calling console.info makes everything tick. Calling it breaks the whole thing, no output on the on-screen div whatsoever.
Edit 4: Culprit found. screenlog.js line 91: 

> console.info = originalFnCallDecorator(_**clear**_, 'info'); 

should read 

> console.info = originalFnCallDecorator(_**info**_, 'info');

Serves me right for copy pasting. :( Could you fix this on your end? I have to make another pull request for drag functionality and maybe another for positioning api.

**Comment by chinchang** [comment]
I'll fix.

**Comment by chinchang** [comment]
fixed in 0.2.1

I think I should put in test asap :)

**Pull request merged**